pubspec.yaml

```diff
@@ -1,20 +1,19 @@
 name: flutter_redux
 description: A set of utility Widgets that Provide and Connect to a Redux Store
-version: 0.7.0
+version: 1.0.0
 author: Brian Egan <brian@brianegan.com>
 homepage: https://github.com/brianegan/flutter_redux
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  meta: ">=1.1.1 <2.0.0"
-  redux: ">=4.0.0 <5.0.0"
+  redux: ">=5.0.0 <6.0.0"
   flutter:
     sdk: flutter
 
 dev_dependencies:
-  pedantic: 1.8.0+1
+  pedantic: 1.10.0
   flutter_test:
     sdk: flutter
-  mockito: 4.1.1
+  mockito: 5.0.0
```

Comment:
I understand that the currently referenced mockito version is specific and so the new one might as well be, but I'm also nervous about committing to a point 0 release. Would it be so bad to be flexible here?
`mockito: ^5.0.0`

Reply:
I added answer to another your comment